packages/taquito-rpc/src/rpc-client-interface.ts

```diff
@@ -127,6 +128,7 @@ export interface RpcClientInterface {
     options?: RPCOptions
   ): Promise<string>;
   getAllTicketBalances(contract: string, options?: RPCOptions): Promise<AllTicketBalances>;
+  getOriginationProof(reqs: OriginationProofParams, options?: RPCOptions): Promise<string>;
```

Comment:
With this rpc endpoint added, might want to add a test case in `integration-tests/rpc-nodes.spec.ts`

Reply:
will do thanks :D